auslib/web/admin/swagger/api.yaml

```diff
@@ -108,6 +131,110 @@ parameters:
     minimum: 0
     required: true
 
+  change_id_param:
+    name: change_id
+    in: path
+    description: Unique id to identify the identify the change
+    type: string
+    required: true
+
+  field_name_param:
+    name: field
+    in: path
+    description: name of DB column whose history is requested
+    type: string
+    required: true
+
+  userPermissionFormDataParam:
+    name: user_permission_request_body
+    in: body
+    description: Request body containing options and data version field values for manipulating user permission values in DB
+    required: true
+    schema:
+      allOf:
+        - $ref: '#/definitions/CSRFModel'
+        - type: object
+          properties:
+            data_version:
+              type: integer
+              format: int32
+              description: data_version of permission
+              minimum: 1
+              example: 1
```

Comment:
No , data_version is only needed when updating an existing permission. [Link](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/permissions.py#L76) whereas when creating a new permission it isn't required. [Link](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/permissions.py#L85)